src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -631,7 +631,6 @@ public async Task<IActionResult> RegisterExternalLogin(RegisterExternalLoginView
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> LinkExternalLogin(LinkExternalLoginViewModel model, string returnUrl = null)
         {
-            var settings = (await _siteService.GetSiteSettingsAsync()).As<RegistrationSettings>();
```

Comment:
Oops, it was not used!

Reply:
Yup! Couple places like that so I removed them since I was cleaning up the use `ISiteService`